chainer/functions/normalization/instance_normalization.py

```diff
@@ -0,0 +1,49 @@
+import warnings
+
+from chainer.functions.array import reshape
+from chainer.functions.array import tile
+from chainer.functions.normalization import batch_normalization
+
+
+def instance_normalization(x, gamma, beta, eps=2e-5):
+    """Instance normalization function.
+
+    This function implements a "instance normalization"
```

Comment:
You can remove the `a` and possible simplify it as follows.
```suggestion
    This function implements instance normalization
```